easybuild/easyconfigs/n/netCDF/netCDF-4.6.0-intel-2018a-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,36 @@
+name = 'netCDF'
+version = '4.6.0'
+hdf5_ver = '1.10.1'
+versionsuffix = '-HDF5-%s' % hdf5_ver
```

Comment:
@SimonPinches The existing `netCDF-4.6.0-intel-2018a.eb` has exactly the same `HDF5` dependency as this one, so there's no reason to add this easyconfig with a custom `versionsuffix` (we only do this in case we need the same `netCDF` version with the same toolchain but with a different version of a particular dependency)